packages/core/src/components/steps-modal/steps-modal.tsx

```diff
@@ -76,9 +78,9 @@ export class AtomStepsModal {
     return currentStep === steps ? handleFinish() : handleNextStep()
   }
 
-  private handleCloseClick = () => {
+  private handleCloseClick = (e: Event) => {
+    e.stopImmediatePropagation()
```

Comment:
could we test it too?

Reply:
sure! [508ed2d](https://github.com/juntossomosmais/atomium/pull/609/commits/508ed2dd19c46b1a866caf7985d684b3db0b2205)